resources/content/config-content/bootstrap/bootstrap.sh

```diff
@@ -104,6 +104,11 @@ upgrade()
     fi
 }
 
+get_running_image()
+{
+ echo $(docker inspect rancher-agent|jq -r .[].Config.Image)
```

Comment:
Can you switch to `docker inspect -f '{{.Config.Image}}'` and use four spaces and not a tab